lib/RuntimeNode43.js

```diff
@@ -31,6 +31,12 @@ module.exports = function(S) {
               functionFile    = func.getRootPath(handlerArr[0] + (func.handlerExt || '.js')),
               functionHandler = handlerArr[1];
 
+            // Switch to the root of the lambda package from the handler property
+            const handlerFullPath = func.getRootPath(_.last(func.handler.split('/'))).replace(/\\/g, '/');
```

Comment:
This is definitely intentional as the whole path handling is copy-paste from another location in sls. I guess the whole path handling would need some rework, but that wasn't the intention of this PR.